Realm/ObjectServerTests/RLMSyncTestCase.mm

```diff
@@ -344,13 +344,10 @@ - (RLMRealm *)immediatelyOpenRealmForURL:(NSURL *)url
                                     user:(RLMSyncUser *)user
                            encryptionKey:(NSData *)encryptionKey
                               stopPolicy:(RLMSyncStopPolicy)stopPolicy {
-    auto syncConfig = [[RLMSyncConfiguration alloc] initWithUser:user realmURL:url];
-    syncConfig.stopPolicy = stopPolicy;
-    syncConfig.isPartial = [self isPartial];
-
-    auto c = [RLMRealmConfiguration defaultConfiguration];
-    c.encryptionKey = encryptionKey;
-    c.syncConfiguration = syncConfig;
+    auto c = [user createConfiguration:url];
+    c.encryptionKey = encryptionKey;  
+    c.syncConfiguration.stopPolicy = stopPolicy;
```

Comment:
This doesn't work; the `syncConfiguration` property does not return an object which mutates the RLMRealmConfiguration it's obtained from.